internal/querynode/result.go

```diff
@@ -31,6 +34,45 @@ import (
 	"github.com/milvus-io/milvus/internal/util/typeutil"
 )
 
+func reduceStatisticResponse(results []*internalpb.GetStatisticsResponse) (*internalpb.GetStatisticsResponse, error) {
```

Comment:
Move this function to a common util. How about this `Reduce(maps, methodsOnKey)` and then define `methodsOnKey` outside.